test/libsolidity/semanticTests/functionTypes/comparison_operator_for_external_function_cleans_dirty_bits.sol

```diff
@@ -0,0 +1,16 @@
+contract C {
+  function g() external {}
+  function comparison_operators_for_external_function_pointers_with_dirty_bits() external returns (bool) {
+        function() external g_ptr_dirty = this.g;
+        assembly {
+                g_ptr_dirty.address := or(g_ptr_dirty.address, shl(160, sub(0,1)))
+                g_ptr_dirty.selector := or(g_ptr_dirty.selector, shl(32, sub(0,1)))
+        }
+        function() external g_ptr = this.g;
+        return g_ptr == g_ptr_dirty;
+  }
+}
+// ====
+// compileViaYul: also
```

Comment:
```suggestion
// compileViaYul: also
// EVMVersion: >=constantinople
```
This should fix the remaining test failure in CI. Shift opcodes like ``shl`` are not available in ancient EVM versions, so we can't run this test on them. But since the cleanup code is the same across EVM version only checking for newer EVM versions is just fine for this.